code/modules/atmospherics/machinery/components/binary_devices/volume_pump.dm

```diff
@@ -49,6 +55,10 @@
 	SSradio.remove_object(src,frequency)
 	return ..()
 
+/obj/machinery/atmospherics/components/binary/volume_pump/proc/set_on(active)
+	on = active
+	SEND_SIGNAL(src, COMSIG_PUMP_SET_ON, on)
```

Comment:
Why don't I see this signal being created anywhere? Are you reusing signals? If so, you should not be.

Reply:
Yeah, I was reusing  the gas pump's signal. I'll create a new one.